woocommerce/woocommerce
# Migrate Category field to woocommerce/taxonomy-field block

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Migrate Category field to woocommerce/taxonomy-field block



Closes #40019 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Open the new product editor
2. Go to "Organization" tab
3. Test the "Categories" field:
4. Select from existing categories
5. Create a new category with and without parent
6. Save the product

Do the same scenario for the trunk branch and this branch and write down if you detect any differences in behavior. 

Known (and acceptable) differences:
* Error messages from "create new" dialog now appear at the bottom of the name field
* The "Parent" field at "create new" now shows hierarchy




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
